### ISSUE TITLE
Add 008-300

### ISSUE THREAD

**Issue by yacob841** [oyvindwe/connectlife-ha#167]
Hello, my device code is 008-300, I created the 008-300.yaml file, I then put it in the folder with the rest of the yaml files, reloaded and it's still not working. What steps do I take after creating the yaml?

**Comment by yacob841** [comment]
No... maybe I did it wrong lol. I just looked at file 008-304.yaml that already exists and at the entities of my 008-300.yaml. Saw that a majority of them matched up and only really one thing changed which was the name of the daily energy usage, so I just modified that.

**Comment by oyvindwe** [comment]
Did you resolve this? Otherwise, try to enable debugging logging.

**Comment by zacharee** [comment]
I'm running into this too. I copied the yaml for `008-301` since it seems basically identical, but the integration doesn't recognize it. This is all I see in a debug log:

```
2025-06-15 15:14:40.278 DEBUG (MainThread) [custom_components.connectlife] Unloading ConnectLife
2025-06-15 15:14:48.887 DEBUG (MainThread) [custom_components.connectlife] Setting up ConnectLife
2025-06-15 15:14:48.887 DEBUG (MainThread) [custom_components.connectlife] Options: {}
2025-06-15 15:14:50.826 DEBUG (MainThread) [custom_components.connectlife.coordinator] Finished fetching connectlife data in 0.352 seconds (success: True)
```

**Comment by zacharee** [comment]
Nevermind, apparently it just needed another reboot. There is a `super` mode that this has which isn't in the 301 config, but otherwise it's identical.

**Comment by oyvindwe** [comment]
`t_super` added in PR #419
